nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/NotificationServiceManager.java

```diff
@@ -247,7 +247,7 @@ public PropertyValue getProperty(final PropertyDescriptor descriptor) {
                     configuredValue = fullPropDescriptor.getDefaultValue();
                 }
 
-                return new StandardPropertyValue(configuredValue, null);
+                return new StandardPropertyValue(configuredValue, null, null);
```

Comment:
Didn't appear to be a necessity in previous version however doesn't hurt to add.  Will get it done!